docs/admin/templates/managing-templates/external-workspaces.md

```diff
@@ -0,0 +1,129 @@
+# External workspaces
+
+External workspaces allow you to connect agents running on external infrastructure to your Coder deployment. This feature enables you to bring existing servers, on-premises infrastructure, or any machine that can run the Coder agent into your development environment without requiring Coder to provision the compute resources.
+
+## Prerequisites
+
+- Access to external compute resources (VMs, bare-metal servers, Kubernetes nodes, etc.) that can run the Coder agent.
+- Networking configured so the external agent can reach your Coder deployment.
+- A workspace template that includes a coder_external_agent resource.
+- To import the template and begin configuring it, follow the [documentation in the Coder Registry](https://registry.coder.com/templates/coder-labs/externally-managed-workspace)
```

Comment:
```suggestion


We provide an example template on how to set up external workspaces in the [Coder Registry](https://registry.coder.com/templates/coder-labs/externally-managed-workspace)
```
This isn't a prerequisite but a full working example. Maybe move out of this section.